src/pf-utils/pf-utils.js

```diff
@@ -38,4 +38,76 @@ let I18nUtil = function() {
     self.mixin = mixin;
   };
 };
-export let i18n = new I18nUtil();
+let i18n = new I18nUtil();
+export {i18n};
+
+/**
+ * --------------------------------------------------------------------------
+ * PfUtil
+ * Native Javascript for Bootstrap 3 | Internal Utility Functions
+ * --------------------------------------------------------------------------
+ */
+
+class PfUtil {
+
+  constructor () {
+    this.isIE = (new RegExp("MSIE ([0-9]{1,}[\.0-9]{0,})").exec(navigator.userAgent) !== null) ? parseFloat( RegExp.$1 ) : false;
```

Comment:
This won't detect all versions of IE (e.g., Edge). We're using this in the table JS.

return /(MSIE|Trident\/|Edge\/)/i.test(window.navigator.userAgent);